addon/components/time-series-chart.js

```diff
@@ -970,9 +980,18 @@ const TimeSeriesChartComponent = ChartComponent.extend(LegendMixin,
     this.removeAllSeries();
 
     var series = this.get('series');
+    var showDetails = this.get('showDetails');
+    var hideDetails = this.get('hideDetails');
+    
     series.enter()
       .append('g').attr('class', 'series')
-      .append('path').attr('class', 'line');
+      .append('path').attr('class', 'line')
+        .on("mouseover", function(d, i) {
```

Comment:
I re-wrote large swaths of this stuff and there are still things that I don't quite understand :-P